adhocteam/Head-Start-TTADP
# Cm 302 create mainenance page

**Description of change**
Add a maintenance page to the smart hub. This will allow us to point the ttahub.ohs.acf.hhs.gov domain to this page if we ever need to do maintenance on the app. We will still be able to reach the app through the tta-smarthub-prod.app.cloud.gov route. 

This PR only sets up the infrastructure. Next I will write the script to put the app into and out of maintenance mode and document how to do it.

**How to test**
Verify the page deployed properly to the sandbox environment. You can find it at https://tta-smarthub-maintenance-page-sandbox.app.cloud.gov/.

Verify the page matches the design at tta-smarthub-prod.app.cloud.gov.

Verify that there are no obvious accessibility issues with the page.

**Issue(s)**
* https://github.com/HHS/Head-Start-TTADP/issues/302

**Checklist**

    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified
